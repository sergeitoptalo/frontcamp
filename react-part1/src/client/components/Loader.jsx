import React from 'react';

const Loader = (props) => (
    <div className="bouncing-loader">
        <div></div>
        <div></div>
        <div></div>
    </div>
);

export default Loader;
