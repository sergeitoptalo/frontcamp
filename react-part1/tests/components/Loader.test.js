import React from 'react';
import renderer from 'react-test-renderer';

import Loader from '../../src/client/components/Loader';

it('Should be rendered correctly', () => {
    const component = renderer.create(
        <Loader />
    )
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
