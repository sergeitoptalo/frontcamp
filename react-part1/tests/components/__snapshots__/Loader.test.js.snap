// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be rendered correctly 1`] = `
<div
  className="bouncing-loader"
>
  <div />
  <div />
  <div />
</div>
`;
